packages/linux_sources.rb

```diff
@@ -3,12 +3,17 @@
 class Linux_sources < Package
   description 'Sources for the Linux kernel'
   homepage 'https://kernel.org/'
-  version '3.18'
-  source_url 'https://www.kernel.org/pub/linux/kernel/v3.x/linux-3.18.tar.xz'
-  source_sha256 'becc413cc9e6d7f5cc52a3ce66d65c3725bc1d1cc1001f4ce6c32b69eb188cbd'
+  version '4.14'
+  source_url 'https://www.kernel.org/pub/linux/kernel/v4.x/linux-4.14.tar.xz'
+  source_sha256 'f81d59477e90a130857ce18dc02f4fbe5725854911db1e7ba770c7cd350f96a7'
 
   def self.install
     linux_src_dir = CREW_DEST_PREFIX + '/src/linux'
+    system 'wget', 'https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+archive/refs/heads/release-R75-12105.B/sys-kernel/linux-headers/files.tar.gz'
```

Comment:
I've excluded creating a sha256sum for this due to the fact that the archives are created on-the-fly and change every time they're downloaded.

Reply:
No tagged git repo to pull this from?

Reply:
Nope. Unless you want to have this download the entire Chromium OS overlay (which is a bit much just for some Linux header patches).

Reply:
`git clone -b release-R75-12105.B https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay.git`?

Reply:
The overlay repository is very large and is unneeded if you just need a few patches.

Reply:
@uberhacker Although most people's bandwidth could probably manage if I only downloaded one commit... Does that sound alright?

Reply:
Sure, I just haven't had time to test this out yet...day job. :/

Reply:
@uberhacker Done.

Reply:
![image](https://user-images.githubusercontent.com/38843657/62158055-9989b980-b2dc-11e9-811a-f55fcb93b74a.png)